op-devstack/sysgo/l2_el.go

```diff
@@ -25,9 +27,23 @@ func L2ELWithSupervisor(supervisorID stack.SupervisorID) L2ELOption {
 	})
 }
 
+func L2ELWithListenAddr(listenAddr string) L2ELOption {
+	return L2ELOptionFn(func(p devtest.P, id stack.L2ELNodeID, cfg *L2ELConfig) {
+		cfg.ListenAddr = listenAddr
+	})
+}
+
+func L2ELWithNetrestrictTxPoolGossip(netrestrictTxPoolGossip string) L2ELOption {
+	return L2ELOptionFn(func(p devtest.P, id stack.L2ELNodeID, cfg *L2ELConfig) {
+		cfg.NetrestrictTxPoolGossip = netrestrictTxPoolGossip
+	})
+}
+
 func DefaultL2ELConfig() *L2ELConfig {
 	return &L2ELConfig{
-		SupervisorID: nil,
+		SupervisorID:            nil,
+		ListenAddr:              "127.0.0.1:0",
+		NetrestrictTxPoolGossip: "127.0.0.1/32,127.0.0.2/32",
```

Comment:
Defaulting to `"127.0.0.1/32,127.0.0.2/32"` may potentially break other tests. Can we explicitly define the tx restrict gossiping during `DefaultSingleChainThreeNodesSystem`?

Reply:
Why would it break other tests, given that we start with `ListenAddr: 127.0.0.1`? It shouldn't break any tests unless they modify the `ListenAddr` on purpose, which no tests do.

We can have this configurable, I just would rather not add more code, until it is actually necessary.